dspace/solr/statistics/conf/schema.xml

```diff
@@ -186,158 +60,68 @@
-   <field name="rpp" type="integer" indexed="true" stored="true" required="false" />
-   <field name="sortBy" type="string" indexed="true" stored="true" required="false" docValues="true"/>
-   <field name="sortOrder" type="string" indexed="true" stored="true" required="false" docValues="true"/>
-   <field name="page" type="integer" indexed="true" stored="true" required="false" />
-
-   <!--Workflow spefic-->
-   <field name="workflowStep" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
-   <field name="previousWorkflowStep" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
-   <field name="owner" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
-   <field name="submitter" type="string" indexed="true" stored="true" required="false" />
-   <field name="actor" type="string" indexed="true" stored="true" required="false" />
-   <field name="workflowItemId" type="integer" indexed="true" stored="true" required="false" />
-
+    <field name="uid" type="string" indexed="true" stored="true" required="true" docValues="true"/>
+
+    <!--Can either be view/search/search_result/workflow-->
+    <field name="statistics_type" type="string" indexed="true" stored="true" required="true" default="view" docValues="true"/>
+
+    <!-- Search specific  -->
+    <field name="query" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
+    <field name="scopeType" type="integer" indexed="true" stored="true" required="false" />
+    <field name="scopeId" type="string" indexed="true" stored="true" required="false" />
+    <field name="rpp" type="integer" indexed="true" stored="true" required="false" />
+    <field name="sortBy" type="string" indexed="true" stored="true" required="false" docValues="true"/>
+    <field name="sortOrder" type="string" indexed="true" stored="true" required="false" docValues="true"/>
+    <field name="page" type="integer" indexed="true" stored="true" required="false" />
+
+    <!--Workflow spefic-->
+    <field name="workflowStep" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
+    <field name="previousWorkflowStep" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
+    <field name="owner" type="string" indexed="true" stored="true" required="false" multiValued="true" docValues="true"/>
+    <field name="submitter" type="string" indexed="true" stored="true" required="false" />
+    <field name="actor" type="string" indexed="true" stored="true" required="false" />
+    <field name="workflowItemId" type="integer" indexed="true" stored="true" required="false" />
 
  </fields>
 
  <uniqueKey>uid</uniqueKey>
 
- <!-- field for the QueryParser to use when an explicit fieldname is absent -->
```

Comment:
The defaultSearchField and defaultOperator change removal will need to be tested.  When DSpace used to default the operator to OR, results were not very reliable.

Reply:
Changing these was unintentional, the result of a half-done fix.